tests/src/Exceptions/ForeignThread/ForeignThreadExceptionsNative.cpp

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "stdio.h"
+
+#pragma warning(push)
+#pragma warning(disable:4265 4577)
+ #include <thread>
+#pragma warning(pop)
+
+
+#if defined _WIN32
+  #define DLL_EXPORT __declspec(dllexport)
```

Comment:
I've added it to Common/Platform/platformdefines.h and will use it from there instead.